angular/projects/admin/src/app/admin/settings/settings.component.ts

```diff
@@ -99,6 +120,8 @@ export class SettingsComponent implements OnInit, OnDestroy {
     return this.siteSettingsService.save({
       title: formData.title,
       theme: formData.theme,
+      defaultLanguage: formData.defaultLanguage,
+      languages: this.languages,
```

Comment:
Make sure to sort the array so that default language is on index 0 as the modal is preventing index 0 from being deleted.

Reply:
Solved on this commit https://github.com/oddbit/tanam/pull/51/commits/0e6ff58c763855bb6ba0de84576ed12e9cd14441